### ISSUE TITLE
Feature Request: Could we get a raws only option?

### ISSUE THREAD

**Issue by KAmaia** [Drumstix42/drumstix42.github.io#3]
As the title states, a raws only option would be amazing, so I can tell exactly HOW MANY copper bars will be needed to craft 100 wooden cabinets etc., without having to read the whole list.

**Comment by Drumstix42** [comment]
Yeah I think I could implement that into a section. I've got a few outstanding requests to add improvements to the calculator, and haven't had the time to tackle them. I'll be away for work for a bit, but when I return I may be able to add some of these to the output areas.

Thanks for your suggestion.

**Comment by Drumstix42** [comment]
I know this is pretty old, but I revisited the tool a bit this year to keep it update and wanted to add this improvement.

Now (and by default) the "Raws" are split into their own section. The old functionality can be restored by disabling the new toggle setting.

**Please note** While I tried to cover most of the items that are considered "Raw" materials, I had to make a list manually using exact matches/regex (a sort of wild card matching) to make this work -- as the game doesn't really give me that information from the game files. In short: I very likely missed some Raws, so **feel free to report any missing ones!**

Example:

<img width="553" height="822" alt="Image" src="https://github.com/user-attachments/assets/e30ee511-4e45-4c6a-9e01-475d8142a386" />
